apps/emqx_auth/src/proto/emqx_auth_cache_proto_v1.erl

```diff
@@ -1,5 +1,5 @@
 %%--------------------------------------------------------------------
-%% Copyright (c) 2018-2024 EMQ Technologies Co., Ltd. All Rights Reserved.
+%% Copyright (c) 2024 EMQ Technologies Co., Ltd. All Rights Reserved.
```

Comment:
```suggestion
%% Copyright (c) 2024-2025 EMQ Technologies Co., Ltd. All Rights Reserved.
```

Reply:
The commit that changed the file is from 2024. Do we need to also bump the other files touched by the PR?

Reply:
yes. there is a script to change all files: `./scripts/update-copyright-years.sh`